lib/scss/inc/_base.scss

```diff
@@ -55,7 +55,8 @@ h6 { font-size: $h6-font-size; }
 
 h1, h2, h3, h4, h5, h6 {
   margin: 25px 0 12px 0;
-  font-weight: $font-weight-bold;
+  font-weight: $font-weight-base;
```

Comment:
Following Jayshree's feedback, could we also keep this as `font-weight: $font-weight-bold;`?